utils/defs.tex

```diff
@@ -135,7 +135,7 @@
   }\process@me
 }
 \makeatother
-\newcommand{\ChangelogRef}[1]{\\See \cref{#1}.}
+\newcommand{\ChangelogRef}[1]{\\See \cref{#1}.\space}
```

Comment:
Do you still need `\space` if you remove the `%` immediately after the usage of this macro?

Reply:
Nope. You don't need `\space`, but it doesn't hurt either because the space is elided if there's no subsequent sentence. I'd prefer to have the `%` visual reminder even if it causes this weird case here.